### DISCUSSION TITLE
Show category descriptions somewhere, anywhere

### DISCUSSION THREAD

**Discussion by Griatch** [community/community#2941] [Discussions]
When defining the discussion categories, one get to specify a line of description. This description is then, as far as I can tell _not used anywhere_. Some suggested places where it could be used: 

- It could show as a tool tip when hovering over the categories on the left.
- It could show at top of the list when clicking a particular category.

The description is useful for describing what the category is for. As it is I have to create a new post (which I seemingly cannot pin per-category) in each category to explain what should go there and not.

**Comment by evi-liu** [comment]
Hi @Griatch, thanks for the feedback! You can currently see category descriptions when creating a new discussion, but I totally hear the need to see it on the homepage. Curious for others to chime in here as well to see how important this is! 

![image](https://user-images.githubusercontent.com/8809849/101825995-41bfad00-3ae3-11eb-99bb-a0e9d4c5f084.png)

**Reply by Griatch** [reply]
@evi-liu Ah, thanks, I didn't even notice this. I guess it depends on how one views Discussions - being used to forums I suppose I imagine the categories like subforums. Thinking about it, I tend to expect to click on the category, see what's there and then click `New post` to _write in that category_. But this is not what's going on - the `New post` button is not tied to the category you are in at all, so you have to specify it in that dropdown. 
I can see the reasoning since the category-view only acts as a filter for all posts, it's just a very different way to view things compared to what I'm used to, I guess. I at least tend to want to look through the categories to see what they mean without having to create a post first.

**Reply by jarrodmillman** [reply]
I would like to see the Description displayed when you hover over the category.  Presumably if someone is hovering over the title of the category they are looking for a description.  Currently hovering over the title shows the title.
